### ISSUE TITLE
1.11.2 Crash on startup (similar to the 1.12.2 startup crash)

### ISSUE THREAD

**Issue by TheCorinthians** [MinecraftModDevelopment/EndMetals#16]
Main error component (this may look familiar, lol):
net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from End Metals (endmetals)
Caused by: java.lang.NoClassDefFoundError: com/mcmoddev/lib/init/Materials

Crash Report Link:
https://pastebin.com/YDU91p6Z

I'm sorry that it's an older version... I just updated up from 1.10.2 and I had no issues there...

**Comment by dshadowwolf** [comment]
You are missing BaseMetals - if you were on 1.10.2 you had BaseMetals there but you don't have it with your current version.

**Comment by TheCorinthians** [comment]
Ok, I realized that I'm an idiot... I just looked up the files on CurseForge and BaseMetals is **ALSO** a required library [for 1.11.2 and above] and NOT JUST MMD OreSpawn........ (imma dumdum) Though, I looked back in 1.10.2 and I didn't need BaseMetals there and it wasn't in my modpack, which is probably why I thought that it would be the same in 1.11.2... it clearly wasn't... lmao. I'm sorry for bothering you and thank you so much for the help. How do I delete this thread since the issue has been fixed? Or do I have to do the "close issue" thing? (I'm fairly new to GitHub, lol)
